src/ray/raylet/node_manager.cc

```diff
@@ -448,32 +460,61 @@ void NodeManager::HeartbeatAdded(gcs::AsyncGcsClient *client, const ClientID &cl
   }
 }
 
-void NodeManager::HandleActorCreation(const ActorID &actor_id,
-                                      const std::vector<ActorTableDataT> &data) {
-  RAY_LOG(DEBUG) << "Actor creation notification received: " << actor_id;
+void NodeManager::HandleDisconnectedActor(const ActorID &actor_id, bool was_local) {
+  RAY_LOG(INFO) << "Handle disconnected actor";
```

Comment:
Should this be info level? Seems weird that a remote node will log something if some other node dies.